package.json

```diff
@@ -1,18 +1,25 @@
 {
   "name": "defer.js",
   "description": "JavaScript defer",
-  "version": "0.0.5",
+  "version": "1.0.0",
   "repository": {
     "type": "git",
     "url": "git://github.com/fasterize/defer.js.git"
   },
   "scripts": {
-    "build": "./fasterize_build.sh"
+    "build": "./fasterize_build.sh",
+    "dev:test": "mocha ./runner.js tests/test.js",
+    "test": "npm run build && DEFER_FILE=js_defer-min.js mocha --timeout 10000 ./tests/utils/runner.js tests/test.js"
```

Comment:
I think this test should run the `compare_all` tests too.
btw I was unable to run the `compare_all` tests, I get an error about 
```
Error: Cannot find module 'webpage'
    at Function.Module._resolveFilename (module.js:489:15)
    ....

```